app/services/anc_service/patient_visit_label.rb

```diff
@@ -25,7 +25,7 @@ def print1
 
         @patient.encounters.where(["encounter_datetime >= ? AND encounter_datetime <= ? AND program_id = ?",
           @current_range[0]["START"], @current_range[0]["END"], PROGRAM.id]).collect{|e|
-          encounters[e.encounter_datetime.strftime("%d/%b/%Y")] = {"USER" => PersonName.find_by(person_id: e.creator) }
+          encounters[e.encounter_datetime.strftime("%d/%b/%Y")] = {"USER" => PersonName.find_by(person_id: User.find(e.creator).person_id) }
```

Comment:
We could use provider here instead of the creator, as the creator could be a data entry clerk